### PR TITLE
Physical characteristics of Trappist bodies and KSP configuration files

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: quantities/astronomy.hpp [hunk 1]
```diff
@@ -11,13 +11,15 @@ namespace principia {
 namespace quantities {
 
 // This namespace contains units commonly used in astronomy that are not
-// accepted for use with the SI.
+// accepted for use with the SI.  See "Résolutions de la 29e Assemblée Générale
+// de l’Union Astronomique Internationale de 2015".
```
Comment (eggrobin) [inline quantities/astronomy.hpp]: This is specifically B3, and I do not think that your quoted name is anything other than the name of the downloaded PDF, so
```C++
// See Résolution B3 "Sur les valeurs recommandées de constantes de conversion pour une sélection
// de propriétés solaires et planétaires" adopted at the XXIX General Assembly of the IAU in 2015.

File: quantities/astronomy.hpp [hunk 2]
```diff
@@ -11,13 +11,15 @@ namespace principia {
...
 namespace astronomy {
 
 constexpr Mass   SolarMass     = 1.98855e30 * si::Kilogram;
 constexpr Mass   JupiterMass   = 1.8986e27 * si::Kilogram;
 constexpr Mass   EarthMass     = 5.9742e24 * si::Kilogram;
 constexpr Time   JulianYear    = 365.25 * si::Day;
+constexpr Length EarthRadius   = 6.3781e6 * si::Metre;
```
Comment (eggrobin) [inline quantities/astronomy.hpp]: EarthEquatorialRadius?

File: tools/generate_configuration.cpp [hunk 3]
```diff
@@ -31,35 +44,67 @@ constexpr char proto_txt[] = "proto.txt";
 namespace tools {
 namespace internal_generate_configuration {
 
+std::string NormaliseLength(std::string const& s) {
```
[the remark this answers is not in the document]
Comment (pleroy) [reply tools/generate_configuration.cpp]: Done.

File: tools/generate_configuration.cpp [hunk 4]
```diff
@@ -31,35 +44,67 @@ constexpr char proto_txt[] = "proto.txt";
...
+  if (s.find('R') == std::string::npos) {
+    return s;
+  } else {
+    Length const length = ParseQuantity<Length>(s);
+    std::ostringstream stream;
+    stream << std::scientific
+           << std::setprecision(
+                  std::numeric_limits<double>::max_digits10)
+           << length / Kilo(Metre) << " km";
+    return stream.str();
```
[the remark this answers is not in the document]
Comment (pleroy) [reply tools/generate_configuration.cpp]: Done.

File: quantities/astronomy.hpp (hunk 1 above)
Comment (pleroy) [reply quantities/astronomy.hpp]: Done.

File: quantities/astronomy.hpp (hunk 2 above)
Comment (pleroy) [reply quantities/astronomy.hpp]: Done.

**Pull request merged**